README.md

```diff
@@ -38,6 +38,14 @@ Use `--help` option in order to get the full list of options
 --help        Show this message and exit.
 ```
 
+## Usage with srt-live-transmit
```

Comment:
I suggest to make the header of level 3 and put it under and at the very end of "Script Usage" section.

Reply:
Resolved in https://github.com/koenkarsten/srt-stats-plotting/commit/271e76d50a75d01981afd88951eb9d9b34da9df1